src/XMonad/Main.hs

```diff
@@ -186,24 +193,12 @@ xmonadNoargs initxmc serializedWinset serializedExtstate = do
                  return (fromMaybe fbc_ v)
 
     hSetBuffering stdout NoBuffering
+    xinesc <- getCleanedScreenInfo dpy
```

Comment:
I didn't mean to move this line down.  It was an artifact of sloppy refactoring.  I'll move it back.